### ISSUE TITLE
Chips Demo in Gallery not accessible

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16964]
For the chips with the delete button you can only focus on the delete button. You can't focus the chip text.

The focus frame around the different titles of this page are also to big.

/cc @gspencergoog

**Comment by jonahwilliams** [comment]
In _another app with chips_ I noticed that the entire button is tappable instead of just the X

**Comment by goderbauer** [comment]
@jonahwilliams was that a Flutter app?

**Comment by jonahwilliams** [comment]
I've improved the tap handling of the demo - however we should consider updating the demo to not use the "X" icons.

**Comment by gspencergoog** [comment]
How would that work exactly?  They need to have delete buttons for the demo to function (part of the demo is removing things and seeing the available actions change).  Or do you just mean to literally use a different icon for the delete icon?

**Comment by jonahwilliams** [comment]
From the internal guidelines - chips on touch devices should not show the secondary trailing affordance by default.  I think it is _probably_ okay since it is really just a demo of the capabilities.

**Comment by jonahwilliams** [comment]
This is mostly done, but I have not implemented the semantics for chips with 2 or more actions since they are not supported for accessibility on mobile devices

**Comment by clocksmith** [comment]
The material chips on android have separate semantic nodes for the delete button on all chips (they also show a demo of a chip that is both selectable and deletable)

**Comment by clocksmith** [comment]
Closing since the gallery works as expected. But opened https://github.com/flutter/flutter/issues/48739 for better behavior when there is a delete option
